packages/admin/cms-admin/src/pages/createUsePage.tsx

```diff
@@ -220,6 +220,7 @@ export const createUsePage: CreateUsePage =
                     resolveHasConflict: (data) => {
                         return resolveHasSaveConflict(pageState?.document?.updatedAt, data?.page?.document?.updatedAt);
                     },
+                    skip: saving,
```

Comment:
This works for `useSaveConflictQuery`, but not for `useSaveConflict`. I believe `useSaveConflictQuery` was added by @nsams for an easier transition, but should be replace by `useSaveConflict` in the future. So I think we need some support for pausing the polling in the normal `useSaveConflict` too (out of scope of this PR, but please create a ticket)

Reply:
For reference: https://github.com/vivid-planet/comet/pull/748

Reply:
Okay 👍